test/make.sh

```diff
@@ -126,15 +143,18 @@ function check_trailing_whitespace() {
 
 function generate_docs() {
   echo "Generating markdown docs with terraform-docs"
-  local path
-  while read -r path; do
-    if [[ -e "${path}/README.md" ]]; then
-      # script seem to be designed to work into current directory
-      cd $path && echo "Working in ${path} ..."
-      terraform_docs.sh . && echo Success! || echo "Warning! Exit code: ${?}"
-      cd - >/dev/null
+  local pth helper_dir rval
+  helper_dir="$(pwd)/helpers"
+  while read -r pth; do
+    if [[ -e "${pth}/README.md" ]]; then
+      (cd "${pth}" || return 3; "${helper_dir}"/terraform_docs .;)
```

Comment:
This patch moves terraform_docs.sh from the image to the repository to re-enable local doc generation.  See https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/216/files#r301803363 for the related change in the Docker image.

> This is removed because terraform_docs is being added into the helpers folder of each module until such time as each module is updated to use the lint image to perform lint checks.